content/english/blog/post-oscar-fdl-composer.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "FDL Composer to create a workflow with OSCAR"
+date: 2022-06-06T18:00:00+01:00
+# post image
+image: "images/blog/post-oscar-fdl-composer/1-fdl-composer.png"
+# post type (regular/featured)
+type: "featured"
+# meta description
+description: "Using FDL-Composer tool to create a workflow in OSCAR"
+# post draft
+draft: false
+---
+
+
+FDL-Composer is a tool to implement workflows for OSCAR and SCAR. It creates a YAML file from a graphic designer. We are going to simulate the example [video-process](https://github.com/grycap/oscar/tree/master/examples/video-process)
```

Comment:
Please include links to external tools the first time they are mentioned (FDL-Composer, ffmpeg, etc.)